10up/simple-podcasting
# Feature: Block Conversion

### Description of the Change

Adds the ability for the podcast blocks to be transformed to and from the core audio block.

### Benefits

- Allows the user to go back and forth between block types, if needed/desired.
- Uses WP block registration transforms
- Fixes a couple of other odd issues with block in React

### Possible Drawbacks

None noted so far.

### Verification Process

1. Created a test podcast block and populated
2. Clicked "change block" button on block
3. Confirm that audio appears as an option
4. Choose audio block
5. Confirm that block changed to audio and the data from the populated podcast block is applied to audio block
6. Repeat previous steps with audio block to podcast block

### Checklist:






### Applicable Issues

#47 

### Changelog Entry

#### Fixed
- An issue where React would throw an error relating to keys for list items

#### Added
- Ability for user to transform to/from the podcast and audio blocks
